src/System.Reflection.Extensions/tests/project.json

```diff
@@ -0,0 +1,17 @@
+{
+  "dependencies": {
+    "System.Collections": "4.0.10-beta-22809",
```

Comment:
These should be updated based on Eric's recent change to version numbers, right?